.azure-pipelines/test_plan.py

```diff
@@ -556,9 +632,14 @@ def poll(self, test_plan_id, interval=60, timeout=-1, expected_state=""):
                     build_id=build_id,
                     job_name=job_name
                 ).replace(' ', '_')
-            if args.test_set is None or args.test_set == "":
-                # Use topology as default test set if not passed
-                args.test_set = args.topology
+
+            # For KVM PR test, get test modules from pr_test_scripts.yaml, otherwise use args.scripts
+            if args.platform == "kvm":
```

Comment:
`if args.platform.lower() == "kvm":`?

Reply:
Currently, it's an original design point that can be refined, to decide whether the testbed is virtual or physical.
The "kvm" keyword and the same logic are also used in other places, 
I'm pretty sure we won't pass a "KVM", 
let's keep "kvm" and the same logic here, I will refine the logic in the future.